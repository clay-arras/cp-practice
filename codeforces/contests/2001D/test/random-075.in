1
15
8 18 3 14 11 4 3 14 5 6 24 2 12 15 11
