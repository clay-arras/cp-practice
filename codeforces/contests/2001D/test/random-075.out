12
8 18 14 4 3 5 6 24 2 12 15 11
